docs/topics/Gateway.md

```diff
@@ -464,17 +464,27 @@ Commands are requests made to the gateway socket by a client.
 | [Update Voice State](#DOCS_TOPICS_GATEWAY/update-voice-state)       | joins, moves, or disconnects the client from a voice channel |
 | [Update Presence](#DOCS_TOPICS_GATEWAY/update-presence)             | updates a client's presence                                  |
 
-Events are payloads sent over the socket to a client that correspond to events in Discord.
+Events are payloads sent over the socket to a client. Events with an operation code of 0 regards changes in the Discord state, while 
+those with a higher op code affects only the websocket connection.
 
-###### Gateway Events
+###### Gateway Operation Events
+These events do not have a event type specified, but are identified solely on operation code
+
+| op | name                                                                                | description                                                                                                                      |
+|----|-------------------------------------------------------------------------------------|----------------------------------------------------------------------------------------------------------------------------------|
+| 0  | [Dispatch](#DOCS_TOPICS_GATEWAY/gateway-state-events)                               | the data field (d) contains information about changes to the discord state                                                       |
+| 1  | [Heartbeat](#DOCS_TOPICS_GATEWAY/heartbeat)                                         | you must send a heartbeat to discord                                                                                             |
+| 7  | [Reconnect](#DOCS_TOPICS_GATEWAY/reconnect)                                         | server is going away, client should reconnect to gateway and resume                                                              |
+| 9  | [Invalid Session](#DOCS_TOPICS_GATEWAY/invalid-session)                             | failure response to [Identify](#DOCS_TOPICS_GATEWAY/identify) or [Resume](#DOCS_TOPICS_GATEWAY/resume) or invalid active session |
+| 10 | [Hello](#DOCS_TOPICS_GATEWAY/hello)                                                 | defines the heartbeat interval                                                                                                   |
+| 11 | [Heartbeat ACK](#DOCS_TOPICS_GATEWAY/heartbeat-ack)                                 | heartbeat confirmation                                                                                                           |
+
+###### Gateway State Events
```

Comment:
```suggestion
###### Gateway Events

Events are [Dispatch](...) payloads that correspond to events in Discord.
```

Reply:
What's with the `...`?
Will this be updated with a URL in the future or is that something the Docs generator you use can recognize and handle?